backends/bmv2/CMakeLists.txt

```diff
@@ -74,16 +72,16 @@ add_library(bmv2backend ${BMV2_BACKEND_SRCS})
 add_dependencies(bmv2backend genIR)
 
 build_unified(BMV2_SRCS)
-add_executable(p4c-bm2-ss ${BMV2_SRCS})
-target_link_libraries (p4c-bm2-ss bmv2backend ${P4C_LIBRARIES} ${P4C_LIB_DEPS})
+add_executable(p4c-bmv2 ${BMV2_SRCS})
```

Comment:
Can we keep around a symbolic link or a wrapper script to the old program?
We don't want to break people's toolchains unnecessarily.